### DISCUSSION TITLE
How use GitHub pages for monorepo?

### DISCUSSION THREAD

**Discussion by batazor** [slidevjs/slidev#1777] [Q&A]
I would like to use one repository for multiple presentations, does anyone have any examples of how to do this? Maybe ready GitHub actions?

**Comment by kermanx** [comment]
These may help:

- https://github.com/fguisso/talks
- https://github.com/gabriel-del/slidev
- https://github.com/gabriel-del/slidev
- https://github.com/antfu/talks

**Reply by eric-burel** [reply]
To sum it up how it works:
- you need a script to build slides in a loop
- in current versions of Slidev, you can configure the input folder, the output folder, the base path in files URLs => you need to set that up to your slides markdown file name
- Regarding input folder, I haven't managed to preserve CSS styles when moving slides to a nested folder, so it's better to keep the .md source at the root

Building multiple files at once doesn't work because the base path won't be correct, hence the loop. You can also craft a tiny "index.html" that contains a list of links towards each app (there are probably ways to do something more elaborate for the index).
